test/junos/mocked_data/test_get_users/normal/expected_result.json

```diff
@@ -1 +1 @@
-{"dummy-test": {"password": "$1$waUQAEpO$kSmdRL84F5sCoWKHZ8/kU1", "sshkeys": ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC4pFn+shPwTb2yELO4L7NtQrKOJXNeCl1jel9STXVaGnRAnuc2PXl35vnWmcUq6YbUEcgUTRzzXfmelJKuVJTJIlMXii7h2xkbQp0YZIEs4P8ipwnRBAxFfk/ZcDsN3mjep4/yjN56eorF5xs7zP9HbqbJ1dsqk1p3A/9LIL7l6YewLBCwJj6D+fWSJ0/YW+7oH17Fk2HH+tw0L5PcWLHkwA4t60iXn16qDbIk/ze6jv2hDGdCdz7oYQeCE55CCHOHMJWYfN3jcL4s0qv8/u6Ka1FVkV7iMmro7ChThoV/5snI4Ljf2wKqgHH7TfNaCfpU0WvHAnTs8zhOrGScSrtb mircea@master-roshi", "ssh-dss AAAAB3NzaC1kc3MAAACBAIiXYWuYhY7Wyd3E4JYpT/ilm7Q6G/wLlxPgbx4BWRQL8SbVvP/DVLv+1/4aHk4SvPJmYbcGQbx0tz0pHa3YLmGieLQLVZo44K3IvUXaLf7noUKogwDdGVJ6NRxCROo/uUaU9m8mm4vEQkPgkmrRzNHq1B56AvPmTrRFJc5xPewXAAAAFQDiqOM9bw03MbT4jf59lFSlexRiiQAAAIA0ZJ2dNV9IzeWNexsoQfSz2ABx3INnFvjUAMlWZQtIXAqwUhKAvQBFi6vTpWL9nK4nCUvENyHIgX8S8eG2F1+pRFUHILajiJWkb0+0hGXIgE+rtdsdSCbMzggTOnhr4nMuUqedtas7uwGgOkNoEtsC6uSe0iSf3QbGjyrp8423lAAAAIAR0X7xTJCnV25i14SfelgVztVCl44C/E2A9+EBBR2Dp4CEg3cbstf6Z768S2TxOMpWTr7c81zKI6Ij7qLWxiUlTO7yS3ikiBOpEHbihGeHLn01rHcuGh9+SmOFKeIoq6WJHfZQInO/++WLTtwaZgZzIa1mw18SiFlHFeIX0MfYcA== mircea@master-roshi", "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFAvVeQxnECAx/HcSfcLdNqCqsjdm64QfVdxBT15pmvzw4eV2RWibqZTfCNbybM5FsB0/jEb2EH2JND4GCBzxUg= mircea@master-roshi"], "level": 0}}
+{"dummy-test": {"password": "$1$waUQAEpO$kSmdRL84F5sCoWKHZ8/kU1", "sshkeys": ["ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC4pFn+shPwTb2yELO4L7NtQrKOJXNeCl1jel9STXVaGnRAnuc2PXl35vnWmcUq6YbUEcgUTRzzXfmelJKuVJTJIlMXii7h2xkbQp0YZIEs4P8ipwnRBAxFfk/ZcDsN3mjep4/yjN56eorF5xs7zP9HbqbJ1dsqk1p3A/9LIL7l6YewLBCwJj6D+fWSJ0/YW+7oH17Fk2HH+tw0L5PcWLHkwA4t60iXn16qDbIk/ze6jv2hDGdCdz7oYQeCE55CCHOHMJWYfN3jcL4s0qv8/u6Ka1FVkV7iMmro7ChThoV/5snI4Ljf2wKqgHH7TfNaCfpU0WvHAnTs8zhOrGScSrtb mircea@master-roshi", "ssh-dss AAAAB3NzaC1kc3MAAACBAIiXYWuYhY7Wyd3E4JYpT/ilm7Q6G/wLlxPgbx4BWRQL8SbVvP/DVLv+1/4aHk4SvPJmYbcGQbx0tz0pHa3YLmGieLQLVZo44K3IvUXaLf7noUKogwDdGVJ6NRxCROo/uUaU9m8mm4vEQkPgkmrRzNHq1B56AvPmTrRFJc5xPewXAAAAFQDiqOM9bw03MbT4jf59lFSlexRiiQAAAIA0ZJ2dNV9IzeWNexsoQfSz2ABx3INnFvjUAMlWZQtIXAqwUhKAvQBFi6vTpWL9nK4nCUvENyHIgX8S8eG2F1+pRFUHILajiJWkb0+0hGXIgE+rtdsdSCbMzggTOnhr4nMuUqedtas7uwGgOkNoEtsC6uSe0iSf3QbGjyrp8423lAAAAIAR0X7xTJCnV25i14SfelgVztVCl44C/E2A9+EBBR2Dp4CEg3cbstf6Z768S2TxOMpWTr7c81zKI6Ij7qLWxiUlTO7yS3ikiBOpEHbihGeHLn01rHcuGh9+SmOFKeIoq6WJHfZQInO/++WLTtwaZgZzIa1mw18SiFlHFeIX0MfYcA== mircea@master-roshi", "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFAvVeQxnECAx/HcSfcLdNqCqsjdm64QfVdxBT15pmvzw4eV2RWibqZTfCNbybM5FsB0/jEb2EH2JND4GCBzxUg= mircea@master-roshi"], "level": 0}, u'root': {u'level': 15, u'password': u'$1$LiP5a8I9$feewr345W3H2342', u'sshkeys': []}}
```

Comment:
the bits you added are not valid JSON, that's why the CI is broken.

Reply:
right... was just copy pasting... 
fixed in recent commit 32e5661

Reply:
is that because what I returned is unicode?

Reply:
I removed `u` in the expected result.